builder/amazon/common/run_config.go

```diff
@@ -23,6 +23,7 @@ type RunConfig struct {
 	SourceAmi                string            `mapstructure:"source_ami"`
 	SpotPrice                string            `mapstructure:"spot_price"`
 	SpotPriceAutoProduct     string            `mapstructure:"spot_price_auto_product"`
+	DisableStopInstance      bool              `mapstructure:"disable_stop_instance"`
```

Comment:
Can you add this option to the documentation under `website/` and please include the example use-case that you explained in your PR.